gp-limit-dates/gpld-conditional-limits.php

```diff
@@ -29,6 +29,11 @@ public function init() {
 		add_filter( 'gform_register_init_scripts', array( $this, 'add_init_script' ), 10, 2 );
 		add_filter( 'gform_enqueue_scripts', array( $this, 'enqueue_form_scripts' ) );
 		add_action( 'gform_field_validation', array( $this, 'validate' ), 11, 4 );
+
+        // Enables date limits for the form field, allowing the `gp-limit-dates` script to be enqueued.
+		add_filter( "gpld_has_limit_dates_enabled_{$this->_args['form_id']}_{$this->_args['field_id']}", function ( $result, $field ) {
+			return true;
+		}, 10, 2 );
```

Comment:
```suggestion
        // Enables date limits for the form field, allowing the `gp-limit-dates` script to be enqueued.
		add_filter( "gpld_has_limit_dates_enabled_{$this->_args['form_id']}_{$this->_args['field_id']}", '__return_true', 10, 2 );
```

Seems like a good opportunity for WordPress' built in helper function.